etc/picongpu/hypnos-hzdr/k80_restart.tpl

```diff
@@ -99,31 +99,32 @@ do
         file=""
     fi
 done
+file
```

Comment:
Why is this line added to the script? 
Is this a typo?

Reply:
yes, probably accidentally pasted. not present in the other file